src/gui/foldercreationdialog.cpp

```diff
@@ -10,30 +24,29 @@ namespace  {
   const QString SuggestedFolderNamePrefix = QObject::tr("New folder");
 };
 
-FolderCreationDialog::FolderCreationDialog(QWidget *parent) :
+FolderCreationDialog::FolderCreationDialog(const QString &destination, QWidget *parent) :
```

Comment:
Somehow didn't notice it earlier but we seem to use mostly that convention in our code base (although it's not my favorite):
```
Foo::Foo(QWidget *parent)
    : Base(parent)
    , _member1(...)
    , _member2(...)
{
```